src/app/ReadClient.cpp

```diff
@@ -22,15 +22,55 @@
  *
  */
 
-#include "lib/core/CHIPTLVTypes.h"
 #include <app/AppBuildConfig.h>
 #include <app/InteractionModelEngine.h>
 #include <app/ReadClient.h>
 #include <app/StatusResponse.h>
+#include <lib/core/CHIPTLVTypes.h>
+#include <lib/support/FibonacciUtils.h>
 
 namespace chip {
 namespace app {
 
+/**
+ * @brief The default resubscribe policy will pick a random timeslot
+ * with millisecond resolution over an ever increasing window,
+ * following a fibonacci sequence up to CHIP_RESUBSCRIBE_MAX_FIBONACCI_STEP_INDEX, default is 14
```

Comment:
That "14" should be in the Fibonacci utils header, right?

Reply:
In Fibonacciutil.h there is no CHIP_RESUBSCRIBE_MAX_FIBONACCI_STEP_INDEX, this has been defined CHIPConfig.h, i think i would remove 14 here.

Reply:
The point is, in FibonacciUtils.h there is some boundary where if you ask for that many steps you will not get a useful answer due to overflow.  That boundary should be documented in that header.

Reply:
yep, understood, document with 47, thanks